PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,3 +2,5 @@
 
 Testing: *Describe how this pull request is tested or why it doesn't require tests*
 Fixes: *Link to an issue this pull requests fixes or remove this line if there is no issue*
+
+- [ ] This PR obeys Servo's policy on [AI contributions](https://book.servo.org/contributing.html#ai-contributions)
```

Comment:
Didn't we move away from the checklist-based template because it is too noisy for commit messages?

Reply:
Didn't know that. I'll try me best to comply. What's the best message to convey to remind contributors to obey the policy?

Reply:
> Didn't know that.

Relevant PR: https://github.com/servo/servo/pull/36203

cc @mrobinson who wrote it.

Reply:
I updated the message to be general contribution. It should also help people understand how to open proper PR before create one.